data-egress.tf

```diff
@@ -183,6 +183,26 @@ resource "aws_dynamodb_table_item" "housing_SAS_data_egress_config" {
   ITEM
 }
 
+resource "aws_dynamodb_table_item" "natstats_SAS_data_egress_config" {
+  table_name = aws_dynamodb_table.data_egress.name
+  hash_key   = aws_dynamodb_table.data_egress.hash_key
+  range_key  = aws_dynamodb_table.data_egress.range_key
+
+  item = <<ITEM
+  {
+    "source_prefix":                {"S":    "dataegress/sas/uc_natstats/export/$TODAYS_DATE/*"},
+    "pipeline_name":                {"S":    "DWX-SAS-SFT02"},
+    "recipient_name":               {"S":    "Housing"},
```

Comment:
Not sure this will be right, but we can leave it and see if it causes any problems during the prod test.